.gitignore

```
@@ -32,4 +32,7 @@ node_modules
 /devops/local/data/nodered/
 /devops/local/data/mosquitto/
 
+/devops/local/data/kafka/
+/devops/local/data/zoo/
```

IMO we should do like this, was thinking from long to changes 😉. Please do in case you doing changes here 
```suggestion
/devops/local/data/
```

---

Sure